contracts/facets/EmbalmerFacet.sol

```diff
@@ -177,16 +178,16 @@ contract EmbalmerFacet {
             );
         }
 
-        // Confirm that archaeologists are provided
+        // Validate archaeologist and threshold lengths
         if (selectedArchaeologists.length == 0) {
             revert NoArchaeologistsProvided();
         }
-        // Confirm that threshold is greater than 0
+
         if (sarcophagusParams.threshold == 0) {
             revert ThresholdCannotBeZero();
         }
-        // Confirm that threshold is less than or equal to the number of archaeologists
-        // (k <= n in a shamir secret sharing scheme)
+
```

Comment:
Is there a missing comment here?

Reply:
Nope. This comment is actually probably unnecessary altogether. I'll update it though to read a little less abruptly